elementary/lineage/cli.py

```diff
@@ -127,15 +127,21 @@ def get_cli_lineage_generate_properties() -> dict:
     cls=RequiredIf,
     required_if='profiles_dir'
 )
+@click.option(
```

Comment:
Sorry for asking this, could you please remove all the changes under `elementary/lineage`?
The reason is that this module is deprecated and we'll delete it soon.
Given that, introducing more changes to it will just increase the likelihood of breaking things.

Reply:
done